tss-esapi/src/interface_types/data_handles.rs

```diff
@@ -53,10 +59,118 @@ pub enum Entity {
     Platform,
     Endorsement,
     Lockout,
-    // TODO: Handle Auth
+    // TODO: Handle Auth, that is vendor specific.
 }
 
 #[derive(Debug, Copy, Clone)]
 pub enum Pcr {
     Pcr(PcrTpmHandle),
 }
+
+/// Enum representing the 'Context' data handles interface type.
+///
+/// # Details
+/// This corresponds to the TPMI_DH_CONTEXT interface type.
+/// This corresponds to the TPMI_DH_CONTEXT interface type. This only
```

Comment:
The sentence seems to be repeated twice here.

Reply:
I'd also use backticks in more places: `TPMI_DH_CONTEXT` and maybe "the `Context` data"

Reply:
Fixed